### PR TITLE
Feast: Tooltip & Tracking Link Changes

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What are you doing in this PR?

Fast Tooltip and Tracking Link Chnages specified below:-

- Tooltip copy ->
`Make a feast out of anything with the Guardian’s new recipe app. Feast has thousands of recipes including quick and budget-friendly weeknight dinners, and showstopping weekend dishes – plus smart app features to make mealtimes inspiring.`

- `new benefits` reads `new`

- Thankyou Page Single Android/Ios Download Link (tracking implied) ->
https://guardian-feast.go.link?adj_t=1dufrlhf&adj_fallback=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app&adj_redirect_macos=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app

[**Trello Card**](https://trello.com/c/6NYrax1T/857-wording-and-app-download-changes-update-to-feast-on-lp-ty-page)

## Screenshots
|before|after|
|-----|-----|
|image|image|

Testing
https://support.thegulocal.com/uk/contribute#ab-feast=variant

### CODE REVIEW

File: support-frontend/assets/helpers/urls/externalLinks.ts [hunk 1]
```diff
@@ -32,8 +32,7 @@ const homeDeliveryUrl = `https://www.${getBaseDomain()}/help/2017/dec/11/help-wi
 const defaultIntCmp = 'gdnwb_copts_bundles_landing_default';
 const androidAppUrl =
 	'https://play.google.com/store/apps/details?id=com.guardian';
-const androidFeastAppUrl =
-	'https://play.google.com/store/apps/details?id=uk.co.guardian.feast';
+const feastAppUrl = `https://www.theguardian.com/help/insideguardian/2024/apr/17/introducing-the-feast-app`;
```
Comment (rupertbates) [inline support-frontend/assets/helpers/urls/externalLinks.ts:35]: Shouldn't this be: [https://guardian-feast.go.link?adj_t=1dufrlhf&adj_fallback=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app&adj_redirect_macos=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app](https://guardian-feast.go.link/?adj_t=1dufrlhf&adj_fallback=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app&adj_redirect_macos=https%3A%2F%2Fwww.theguardian.com%2Fhelp%2Finsideguardian%2F2024%2Fapr%2F17%2Fintroducing-the-feast-app)

Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/urls/externalLinks.ts:35]: Hmmm, I thought what it resolved too was more readable (in a format we had in code-base).  Looking again now though without the `guardian-feast.go.link?` tag at the front am guessing the tracking will fail? To be on safe-side will take this string as is. 

Not sure why it has specific dates `2024/apr/17` in the path either (have asked about this direct).
 
Not sure whether we should use a link without tracking for dev/code too (perhaps without `guardian-feast.go.link?`)?

Comment (paul-daniel-dempsey) [reply support-frontend/assets/helpers/urls/externalLinks.ts]: Yep, used full path as suggested.  Done.

File: support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx [hunk 2]
```diff
@@ -228,22 +234,20 @@ export function SupporterPlusThankYou({
...
+		condition: boolean,
 		moduleType: ThankYouModuleType,
-	): ThankYouModuleType[] => (condtion ? [moduleType] : []);
-
-	const showFeast =
-		!!abParticipations.feast &&
-		useContributionsSelector(isSupporterPlusFromState);
+	): ThankYouModuleType[] => (condition ? [moduleType] : []);
 
+	// abParticipation, upon refresh, defaults to active abtTests only
```
Comment (andrewHEguardian) [inline support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx:241]: I don't really understand this comment, could you explain a little more?

**Review by andrewHEguardian — APPROVED** [review]
Nice! 🧑‍🍳

File: support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx (hunk 2 above)
Comment (paul-daniel-dempsey) [reply support-frontend/assets/pages/supporter-plus-thank-you/supporterPlusThankYou.tsx:241]: If you reload the thankyou page, only ab-tests that are marked as `isActive=true` run (including `feast` abtest, which means the Feast card will hide, when the ab-test goes live (ie `isActive=true`) the Feast card will remain visible)

**Pull request merged**